pkg/injector/webhook_test.go

```diff
@@ -534,7 +534,7 @@ var _ = Describe("Testing Injector Functions", func() {
 		}
 
 		// Action !!
-		actual := wh.mutate(nil)
+		actual, _ := wh.mutate(nil)
```

Comment:
please test the additional values being returned in the same or new test.

Reply:
This test just enters the function and immediately exits at the very first condition (nil request). Didn't bother as it is not relevant at all in this instance.

Reply:
> its hard to reason from the code exactly what's going on.

Let's see if I can help.
- on mutate handler, we read timeout from the url.
- if reading the url succeeded, we defer a time-tracking function that will run when `mutateHandler` is done and will print how much time we spent in the handler itself.

Most of the ... ugly code is getting the podname. We could obviate that code and it'd look way cleaner, but the log message would be less relevant without knowing which the resource.